Display focus in the student index

AynAhmed opened the pull request:
### Changes
Give a short summary of the changes made in this PR. 
Made the student's focus show in the table next to the student's name 

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1207258308427302/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 


### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
<img width="504" alt="Screenshot 2024-05-10 at 4 44 25 PM" src="https://github.com/softwareforgood/innocent_tech_app/assets/105397672/4f4145fd-0946-47c8-9326-ac762279e023">


### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

edithemmings commented on app/views/students/index.html.erb:
```diff
@@ -12,6 +12,12 @@
+                <% if active_student_focus %>
+                  <td class="border-b px-4 py-2"><%= active_student_focus.focus.name %></td>
+                <% else %>
+                  <td class="border-b px-4 py-2"></td>
```
Since the contents of the `td` is the only thing that changes, you could move your `if` statement inside of it, and then you only have to define the classes in one place.
This is a good opportunity for a [ternary operator](https://www.rubyguides.com/2019/10/ruby-ternary-operator/) too: 
 <td class="border-b px-4 py-2"><%= @active_student_focus ? active_student_focus.focus.name : '' %></td>
 
Or if you do the nil safety/find the focus itself in the controller, then:
` <td class="border-b px-4 py-2"><%= @focus&.name %></td>`

fouziiah reviewed (approved):
The student focus is showing on my table, looks good  👍

edithemmings reviewed (approved):
Looks awesome! Fantastic job addressing the comments! This is good to go 👍

For future reference, the next level up here is to eager-load associations rather than query them in a loop (also called n + 1 querying). If you'd like to read a little more about this, feel free to explore this section of the Active Record Query docs: https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations

AynAhmed commented:
> Looks awesome! Fantastic job addressing the comments! This is good to go 👍
> 
> For future reference, the next level up here is to eager-load associations rather than query them in a loop (also called n + 1 querying). If you'd like to read a little more about this, feel free to explore this section of the Active Record Query docs: https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations

Thanks! i look into it.
